### PR TITLE
Image tutorial review

### PR DESCRIPTION
**Pull request by Chris-Schnaufer**
Updated to reflect discussed changes

### CODE REVIEW

File: sensors/06-list-datasets-by-plot.Rmd [hunk 1]
```diff
@@ -1,20 +1,23 @@
 # Generating file lists by plot
 
-The terrautils python package has a new products module that aid in connecting
-plot boundaries stored within betydb with file-based data products available
+**Note:** you may need to get access permissions to perform the steps
```
Comment (dlebauer) [inline sensors/06-list-datasets-by-plot.Rmd]:
```suggestion
## Pre-requisites: 

* if you have not already done so, you will need to 1) sign up for the [beta user program](terraref.org/beta) and 2) sign up and be approved for access to the the [sensor data portal](terraref.ncsa.illinois.edu/clowder) in order to get the API key that will be used in this tutorial
```

File: sensors/06-list-datasets-by-plot.Rmd [hunk 2]
```diff
@@ -66,76 +58,109 @@ files = extract_file_paths(datasets)
...
+In the examples below we will be using **curl** on the command line to make our API calls. Since the
+API is accessed through URLs, it's possible to use the URLs in software programs or with a programming language
+to retrieve its data. 
 
-The following method demonstrates the same approach using the Clowder API. This
-approach is useful for understanding the data layout and when the Python
-terrautils package is not available.
+## A Word of Caution
+
+The names of variables in this section don't necessarily match the ones in the section
```
Comment (dlebauer) [inline sensors/06-list-datasets-by-plot.Rmd]: Hmmm ... not sure how to best put this. Explain that in this section you are querying the API directly. Unlike the terrautils client, the API endpoints have names based on the database structure.

Comment (Chris-Schnaufer) [reply sensors/06-list-datasets-by-plot.Rmd]: Made changes

**Comment by dlebauer** [comment]
Lets merge #87, then this one, and we can revisit improvements after we finish the vignettes.

**Comment by KristinaRiemer** [comment]
@Chris-Schnaufer I just want to let you know that I've been working on this, but I'm having a lot of difficulties getting terrautils installed and usable.

**Comment by Chris-Schnaufer** [comment]
From a command line you can run ‘pip install terrautils’. You may need to install ‘pip’ as well.

If it’s installed but not available in R Studio the problem may be the version of Python R Studio  is pointing to. It needs to be the same version that installed terrautils.

On Jan 28, 2019, at 15:12, Kristina Riemer <notifications@github.com<mailto:notifications@github.com>> wrote:


@Chris-Schnaufer<https://github.com/Chris-Schnaufer> I just want to let you know that I've been working on this, but I'm having a lot of difficulties getting terrautils installed and usable.

**Comment by dlebauer** [comment]
Does it help if you set `knitr::opts_chunk$set(engine.path = list(python = 'python3')` or /usr/local/bin/python3 or similar?

**Comment by KristinaRiemer** [comment]
When I used `pip` to try to install on my local machine, there were a lot of errors that seemed to be related to having the newest version of Python (3.7). I was able to get terrautils installed on the Workbench RStudio, but now I'm working on getting gdal. It's pretty normal installation challenges, I just wanted you to know why I'm taking so long.

**Comment by dlebauer** [comment]
these are the errors Kristina mentioned in https://github.com/terraref/tutorials/pull/84#issuecomment-458340058:

```sh
pip install terrautils # works except a few issues w/ influxdb and pyclowder incompatibility
python
import terrautils.products # gives error
```
gives this error:

> from osgeo import ogr
> ImportError: No module named osgeo 

```
pip install osgeo
``` 

returns

> Collecting osgeo
>   Could not find a version that satisfies the requirement osgeo (from versions: )
> No matching distribution found for osgeo

**Comment by Chris-Schnaufer** [comment]
Kristina, let's get together and solve this tomorrow (Tuesday).

________________________________

**Pull request closed without merging**